web/src/components/TwoFactorAuth/index.tsx

```diff
@@ -69,7 +69,7 @@ export const TwoFactorAuthComponent: React.FC<TwoFactorAuthProps> = ({
                                 'cr-email-form__group--focused': codeFocused,
                             })}>
                             <CustomInput
-                                type="number"
+                                type="text"
```

Comment:
we have a new 2fa form input on beneficiary submit modal. Could you please find that component and reuse it here. It looks like the 2fa input in yellow

Reply:
pushed the code.